src/config.toml

```
@@ -1,18 +1,15 @@
 
 [server]
-address = "127.0.0.1:4000"
-file_server_path = "static"
-
-[server.endpoints]
-pod_data = "/podDataStructure"
-order_data = "/orderStructures"
-websocket = "/backend"
-file_server = "/"
+#address = "192.168.0.9:4000"
+address = "127.0.0.9:4000"
+file_server_path = "static" # Change name to front build path
+endpoints = { pod_data = "/podDataStructure", order_data = "/orderDataStructure", websocket = "/backend", file_server = "/" }
```

I find this inline table too long, better make it its own table like before
```suggestion
[server.endpoints]
pod_data = "/podDataStructure"
order_data = "/orderStructures"
websocket = "/backend"
file_server = "/"
```

---

I'll introduce this change in the next PR